configuration/pih/liquibase/liquibase.xml

```diff
@@ -972,4 +962,14 @@
         </sql>
     </changeSet>
 
+    <changeSet id="20240826-migrate-noncoded-dxs-2" author="ddesimone_mgoodrich" runAlways="true" >
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                select count(*) from address_hierarchy_entry where parent_id is NULL and name like 'Haiti';
+            </sqlCheck>
+        </preConditions>
+        <comment>UHM-7593: Migrate ZL MCH non-coded to coded diagnoses (January 2024, updated August 2024)</comment>
+        <sqlFile endDelimiter=";" path="sql/migrate_non_coded_mch_dxs.sql" relativeToChangelogFile="true" stripComments="true"/>
```

Comment:
If we are going to re-run this always, I'm not sure whether the name of the file that is being executed can be changed or not, or if that leads to a different checksum or whatever.  We might want to rename this thing to `migrate_non_coded_dxs.sql` and take the `mch` out of it, as we will likely want to evolve this to cover a broader set of non-coded dx over time.